ledger/storage/jet.go

```diff
@@ -224,3 +228,65 @@ func (db *DB) GetJets(ctx context.Context) (jet.IDSet, error) {
 
 	return jets, nil
 }
+
+func dropSizesPrefixKey() []byte {
```

Comment:
Is this necessary? It's not used in the current code base. We should probably stick to it.